src/live_restore/live_restore_fs.c

```diff
@@ -1581,6 +1592,58 @@ __validate_live_restore_path(WT_FILE_SYSTEM *fs, WT_SESSION_IMPL *session, const
     return (fh->close(fh, (WT_SESSION *)session));
 }
 
+/*
+ * __wt_live_restore_fs_log_copy --
+ *     Copy the log files from the source to the destination prior to recovery. We also need to copy
+ *     over prep log files as logging will expect these are available to be used. This needs to
+ *     happen after __wt_logmgr_config to ensure the relevant logging configuration has been parsed.
+ */
+int
+__wt_live_restore_fs_log_copy(WT_SESSION_IMPL *session)
```

Comment:
Would you want to call it something more general like `wt_live_restore_prepare_recovery` or something like that so that it still makes sense if other things need to get done for LR before recovery that isn't just log files?

Reply:
We could always change that if indeed there are other steps in the future?

Reply:
I will update the function name to something like `live_restore_setup_recovery` and also log the printf message from here using __wt_verbose.

Reply:
05e7091d5